apps/opik-backend/src/test/java/com/comet/opik/infrastructure/auth/RemoteAuthTestServer.java

```diff
@@ -0,0 +1,65 @@
+package com.comet.opik.infrastructure.auth;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.apache.hc.core5.http.HttpStatus;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+
+@NoArgsConstructor
+class RemoteAuthTestServer {
```

Comment:
I'm not against this approach, but probably it's more lightweight to create a mock resource, that can take mock and use the dropwizard extension.

Reply:
We can use WireMock for it

Reply:
changed to wiremock in https://github.com/comet-ml/opik/pull/754/commits/57dea2355bdcb6cf1187fb28759d8b37d2c70fae
I wasn't aware it has this feature set